tree.go

```diff
@@ -88,8 +88,8 @@ func (t Tree) binarySearch(name string) (int, *Node, error) {
 	return pos, nil, ErrNodeNotFound
 }
 
-func (t Tree) Find(name string) (*Node, error) {
-	_, node, err := t.binarySearch(name)
+func (tr Tree) Find(name string) (*Node, error) {
```

Comment:
exported method Tree.Find should have comment or be unexported<br>receiver name tr should be consistent with previous receiver name t for Tree